donut.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+(function() {
+  var DonutShop = function (minCus, maxCus, average, store, open, close) {
+    this.minCus = minCus;
+    this.maxCus = maxCus;
+    this.average = average;
+    this.store = store;
+    this.open = open || 700;
+    this.close = close || 1800;
+    this.hoursOpen = (this.close - this.open)/100;
+    this.hourTotal = [];
+  };
+
+  DonutShop.prototype.donutperhour = function () {
```

Comment:
Good use of prototypes for your methods; i.e. removing those methods from the constructor.